src/predifinedTricksCombos.js

```diff
@@ -0,0 +1,179 @@
+10155,Chrysalis,Frontroll from Feet to Feet,Nick Day,2021,https://www.youtube.com/watch?v=lLCf6JFccwM,,,STAND,STAND,9,In one bounce do a Frontroll starting from Feet and landing back to Feet.,Longer lines allow for more time to execute the trick.
+10156,Turtle Roll,Double Yoda Roll,Pablo Signoret,,https://www.youtube.com/watch?v=VP49hErTqM0&t=420,420,423,EXPOSURE,EXPOSURE,9,Two Yoda Rolls in one bounce.,
+10157,,Fake 360 Front Almighty Flip,Sebastian Egger,2021,https://www.instagram.com/p/COPXx5vtOBy/,,,STAND,STAND,9,Starting in switch stance you do a 270 spin and go into a Front Almighty Flip in the end turn 90 to look back at the same anchor where you started.,
+10158,Overflow,270 Yoda Roll Fake 90,Tim Odermatt,2021,https://www.instagram.com/p/CSHcZfUFG9H/,,,STAND,STAND,9,From your regular stance do a frontside 270 and roll around the line like you would for a Flowflip. Try to land back in your normal stance and not in Exposure.,
+10159,,Makako to Feet,David Palomo,2021,https://www.instagram.com/p/CPDdueLFBro/,,,NEVERMIND,EXPOSURE,10,Makako landing in Exposure.,
+10160,,Despedido to Feet,David Palomo,2021,https://youtu.be/HecUpHXRpDw?t=162,162,178,KOREAN,EXPOSURE,10,Despedido landing in Exposure,
+10161,,180 Super Sonic Mount,Ian Eisenberg,2021,https://youtu.be/HecUpHXRpDw?t=249,249,265,HANG,EXPOSURE,10,It's like a Rocket Rewind to Feet.,
+10162,,360 Front Almighty Flip,Ian Eisenberg,2021,https://youtu.be/HecUpHXRpDw?t=367,367,380,STAND,STAND,10,The real 360 Front Almighty with the starting and ending stance being the same.,
+10163,Reverse Super Sonic Mount,Rocket Back Roll to Feet,Samuel Volery,2021,https://www.youtube.com/watch?v=EntnbHe_biY,,,HANG,EXPOSURE,10,From hanging get a lot of momentum with your feet and do one back roll around the line and land in exposure.,
+10164,,Triple Back Yoda Roll,Davis Hermes,2021,https://www.instagram.com/p/CUNMWEgAGP4/,,,EXPOSURE,EXPOSURE,11,Three Back Yoda Rolls in one bounce.,
+10165,Holy Grail,Barrel Almighty Flip to Barrel Roll,Davis Hermes,2021,https://youtu.be/UdG4DNI0SNk?t=92,92,97,STAND,STAND,11,Start by doing a Barrel Almighty Flip and go for a second roll by grabbing the line and doing a Barrel Roll.,
+10166,,Darth Vader from Feet,Ian Eisenberg,2021,https://youtu.be/HecUpHXRpDw?t=436,436,452,EXPOSURE,EXPOSURE,11,Start in Expo and roll down until you are under the line. Then do a Darth Vader.,
+10167,,Double Back Almighty Flip,Sebastian Egger,2021,https://youtu.be/HecUpHXRpDw?t=547,547,567,EXPOSURE,EXPOSURE,11,Double Back Almighty Flip,Do the first one laid out and tuck for the second one to get good momentum.; Try to get the hook at your ankle. If the line is near your toes it's hard to pull it around for the second roll.
+10168,,Double Front Almighty Flip,Sebastian Egger,2021,https://youtu.be/HecUpHXRpDw?t=780,780,814,EXPOSURE,EXPOSURE,11,Double Front Almighty Flip.,Stay open for the first one and tuck for the second one to get good momentum.`;
+
```

Comment:
Should I put the same ids in the google sheet?

Reply:
Sorry for the two approvals!

Reply:
> Should I put the same ids in the google sheet?

yes that would be awesome

Reply:
> > Should I put the same ids in the google sheet?
> 
> yes that would be awesome

Done!